tendrl/alerting/notification/__init__.py

```diff
@@ -68,15 +64,14 @@ def load_plugins(self):
                 for name, cls in clsmembers:
                     exec("from %s import %s" % (plugin_name, name))
         except (SyntaxError, ValueError, ImportError) as ex:
-            Event(
-                Message(
-                    "error",
-                    "alerting",
-                    {
-                        "message": 'Failed to load the time series db'
-                        'plugins. Error %s' % ex
-                    }
-                )
+            ExceptionMessage(
```

Comment:
@anmolbabu @shtripat first Exception class is created and then it is posted into socket by Event class as a json_str then after received by handler in node-agent  json_str is converted to Message class and then logged. 
Here use of exception class is find traceback of the exception and format the exception But finally Message class object is logged.